lib/betagouvbot/format_mail.rb

```diff
@@ -15,7 +15,7 @@ def initialize(subject, body_t, recipients, sender)
     # @note Email data files consist of 1 subject line plus body
     def self.from_file(body_path, recipients = [], sender = 'secretariat@beta.gouv.fr')
       subject, *rest = File.readlines(body_path)
-      new(subject.strip, rest.join, recipients, sender)
+      new(subject.to_s.strip, rest.join, recipients, sender)
     end
 
     def call(context)
```

Comment:
Just FTR, what I'd have done in a functional language:

```elixir
defmodule MailFormatter do
  def call(:recipients, context) do: blah
  def call(:message, context) do: blahbhabha
end
```

Which in Ruby we would have to write like:

```ruby
class MailFormatter
  class Base
    # ...
   end

  class Recipients < Base
    def call(context); blah; end
  end

  class Message < Base
    def call(context); blahblah; end
  end
end
```

which is, sadly, too verbose and ugly.

My preferred alternative in Ruby would be:

```ruby
class MailFormatter # note the class name change
  def format_recipients(context); blah; end
  def format_message(context) blahblah; end
end
```